test/inductor/test_torchinductor.py

```diff
@@ -7008,6 +7008,24 @@ def fn(a):
             with self.assertRaises(RuntimeError):
                 torch.compile(fn)(a)
 
+        def test_getitem(self):
```

Comment:
Is it common for both CPU and GPU? If it is common, can you move it outside of CPU repro?

Reply:
Done.